tests/clients/detectionlists/test_high_risk_employee.py

```diff
@@ -30,7 +31,7 @@ def mock_detection_list_user_client(self, mock_session, user_context, py42_respo
 
     @pytest.fixture
     def mock_user_client(self, mock_session, user_context, py42_response):
-        user_client = DetectionListUserClient(mock_session, user_context)
+        user_client = UserClient(mock_session)
```

Comment:
A copy paste error in HRE, intended usage was to create UserClient instance here.